tests/test-options.php

```diff
@@ -48,4 +48,14 @@ public function test_delete_options_any_exists() {
 
 	}
 
+	/**
+	 * Test that update_option changes the value of an option that has a default value.
+	 * The 'home' option has a default value. See Options::get_default_options.
+	 */
+	public function test_update_option_with_default_value() {
+		$new_url = 'http://coolsite.com';
```

Comment:
```suggestion
		$new_url = 'https://example.com';
```

Please consider using non-public domain names.

Reply:
Good point. Fixed in commit 8410ff9.

Reply:
Thank you.